### ISSUE TITLE
pulse listener stuck in an endless loop getting tasks for a group

### ISSUE THREAD

**Issue by marco-c** [mozilla/code-coverage#179]
Excerpt from Heroku logs:
```
2019-09-19T00:01:20.256199+00:00 app[worker.1]: 2019-09-19 00:01:20.255993 [INFO    ] code_coverage_events.workflow: Checking code coverage tasks (group_id='SIlcBZkkT3e-8ZFxVK4nwQ' nb=35)

2019-09-19T00:01:21.971862+00:00 app[worker.1]: 2019-09-19 00:01:21.971688 [INFO    ] code_coverage_events.workflow: Checking code coverage tasks (group_id='SIlcBZkkT3e-8ZFxVK4nwQ' nb=200)

2019-09-19T00:01:22.937411+00:00 app[worker.1]: 2019-09-19 00:01:22.937257 [INFO    ] code_coverage_events.workflow: Checking code coverage tasks (group_id='SIlcBZkkT3e-8ZFxVK4nwQ' nb=35)

2019-09-19T00:01:24.491568+00:00 app[worker.1]: 2019-09-19 00:01:24.491402 [INFO    ] code_coverage_events.workflow: Checking code coverage tasks (group_id='SIlcBZkkT3e-8ZFxVK4nwQ' nb=200)

2019-09-19T00:01:25.403826+00:00 app[worker.1]: 2019-09-19 00:01:25.403672 [INFO    ] code_coverage_events.workflow: Checking code coverage tasks (group_id='SIlcBZkkT3e-8ZFxVK4nwQ' nb=35)

2019-09-19T00:01:26.982433+00:00 app[worker.1]: 2019-09-19 00:01:26.982278 [INFO    ] code_coverage_events.workflow: Checking code coverage tasks (group_id='SIlcBZkkT3e-8ZFxVK4nwQ' nb=200)

2019-09-19T00:01:28.029306+00:00 app[worker.1]: 2019-09-19 00:01:28.029144 [INFO    ] code_coverage_events.workflow: Checking code coverage tasks (group_id='SIlcBZkkT3e-8ZFxVK4nwQ' nb=35)
```

Maybe a regression from https://github.com/mozilla/release-services/pull/2244/commits/324d1040f61908543030965d56804dfea26151cb?

**Comment by La0** [comment]
Never saw that one before :'( It's still stuck on that pulse message after a restart

**Comment by La0** [comment]
It looks like a data bug on the pulse queue.
When i listen for messages on [pulse inspector](https://tools.taskcluster.net/pulse-inspector?bindings%5B0%5D%5Bexchange%5D=exchange%2Ftaskcluster-queue%2Fv1%2Ftask-group-resolved&bindings%5B0%5D%5BroutingKeyPattern%5D=%23) i get a lot of messages with the same payload for task group SIlcBZkkT3e-8ZFxVK4nwQ

**Comment by marco-c** [comment]
Yeah, seems to be a Taskcluster bug...
Anyway, I've noticed that the routingKey for this one is primary.SIlcBZkkT3e-8ZFxVK4nwQ.comm-level-1._, schedulerId:comm-level-1.
We could skip these kinds of groups directly and avoid listing the task group.

**Comment by marco-c** [comment]
> Anyway, I've noticed that the routingKey for this one is primary.SIlcBZkkT3e-8ZFxVK4nwQ.comm-level-1._, schedulerId:comm-level-1.
We could skip these kinds of groups directly and avoid listing the task group.

This is an unrelated improvement though, I'll file a separate issue.

**Comment by La0** [comment]
Back to normal as @djmitche published a patch (see [Bug 1582376](https://bugzilla.mozilla.org/show_bug.cgi?id=1582376)).

That |code coverage task](https://tools.taskcluster.net/groups/BIJueExrQXubKyRYuEup4g) was succesfully triggered on testing.

**Comment by La0** [comment]
Closing as this dis not happen again.
